src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -39,6 +39,7 @@ def set_enabled(self, value):
     ### APPLICATOR
 
     def set_bounds(self, x, y, width, height):
+        # print("SET BOUNDS ON", self.interface, x, y, width, height)
```

Comment:
@freakboy3742 would not it be better to start using the logging module for this ?
```
log.debug(("SET BOUNDS ON", self.interface, x, y, width, height))
```

?

Reply:
I get what you're driving at; however, I've got three reasons for not using logging.

1. This is a set of messages that we really only care about during this initial debugging period for Pack. Once we've established that Pack is doing the right thing, we're unlikely to need to have debugging any more.

2. This is a code path that needs to be *fast*. Python function calls are one of the slower operations the VM can perform, so even if a function is a no-op, it can have a perceivable time lag.

3. I have a bit of a love-hate relationship with Log, mostly because it's really easy to use the wrong way. If it's used properly, it's great - but if it isn't... well, you spend your life trying to overcome the fact that some third party package is trying to wrest control of the logging configuration.